documentation/man/features/kw-mail.rst

```diff
@@ -4,6 +4,10 @@ kw-mail
 
 .. _mail-doc:
 
+-------
+kw mail
```

Comment:
I think it is hard to summarize `kw mail`, but maybe something more specific like 'Send patches through mail'